### DISCUSSION TITLE
Cannot find a user who is defined...

### DISCUSSION THREAD

**Discussion by DDEGF** [kimai/kimai#5191] [General] [answered]
I'm an Admin in our Kimai system free plan. I already defined several new users with no issue.  A couple of weeks ago, I defined a new user befor he actually started to work. He started today, was able to login, to input his work hours, and I can see his entry in the "Timesheets"  - BUT ...: somehow I cannot see him in the users list ...- and I need to add some info in his profile.... Any idea ? What is / am I missing...??  Thank you!

**Comment by kevinpapst** [comment, marked as answer]
As written in the email ...
https://www.kimai.org/documentation/users.html#system-accounts

**Reply by DDEGF** [reply]
Indeed. Thank you very much ! 
(was just confusing that an inactive user is able to log entries)

**Reply by kevinpapst** [reply]
An inactive user or a system account? An inactive user shouldn't be able to use Kimai.

**Comment by DDEGF** [comment]
Indeed , thats' why I was confused, but this was the situation:
The user (Leia Dorr) was inactive as I was defined her proactively before
she actually started, - therefore  did not show in the User list,
but was when she started yesterday, someone gave her the link to Kimai and
she was able to log the entry, I saw that in the "Timesheets".

Anyway, I activated her now and all is OK.
Thank you!

**Reply by kevinpapst** [reply]
This story is fishy. An inactive user cannot login.
If a user is active and logged-in and an admin inactivates the account, the user is immediately logged out.
Maybe someone else recorded the timesheet for that account.

**Comment by DDEGF** [comment]
No idea..Not me., not the other Admin.
if someone else did so, it was able to enter Leia as user in the entry.
Anyway, all is Ok now, thank you!

**Reply by kevinpapst** [reply]
Yes. For you its okay.

But I cannot let it stand in public that an inactive user was able to create timesheets.
